tests/upgrades/test_syncplan.py

```diff
@@ -151,21 +156,23 @@ def test_post_disabled_sync_plan_logic(self, request, dependent_scenario_name):
             2. Check the all available sync_interval type update with pre-created sync_plan.
 
         :expectedresults: Update proceedes without any errors. After upgrade, the sync plan
-        should remain the same with all entities.
+        should remain the same with all target_sat.api.
```

Comment:
I see this not only here. Can you please leave entities there?
```suggestion
        should remain the same with all entities.
```

Reply:
that was `:/s/entities/target_sat.api`